Core/src/game/functions/ints/count/Count.java

```diff
@@ -341,6 +348,67 @@ public final class Count extends BaseIntFunction
 	
 	//-------------------------------------------------------------------------
 
+	/**
+	 * For counting elements on a platform below another element
+	 * 
+	 * @param countType The property to count.
+	 * @param type     	The graph element type [default SiteType of the board].
+	 * @param who  		Player id the counted items belong to
+	 * @param site      The site to test.
```

Comment:
please fix the javadoc (it has documentation for more params than there actually are, and in a different order)